internal/resourcestatus/resourcekey_test.go

```diff
@@ -116,7 +116,7 @@ var _ = Describe("Resourcekey", func() {
 	})
 
 	It("returns a list", func() {
-		SetResources(list, &gitrepo.Status.StatusBase)
+		SetGitRepoResources(list, &gitrepo.Status)
```

Comment:
I think it could make sense to validate the `gitrepo`'s `PerClusterResourceCounts` here. How do you see it?